### PR TITLE
MR classification updates

### PR DESCRIPTION
**Pull request by mhedehus**
- Check acquisition label before SeriesDescription, as the label is more likely to be curated by a CRO, and the SeriesDescription is often blank

- Added SWI as label

- Fix T2, T2* problem, acq got classified as both

### CODE REVIEW

**Comment by jiavila** [comment]
@NPann fyi: Going to create another PR to add some edits to the PET classifier

**Review by NPann — APPROVED** [review]
LGTM. Adding a few tests would be nice to cover added classification.

File: MR_classifier.py [hunk 1]
```diff
@@ -576,24 +633,39 @@ def classify_dicom(dcm, slice_number, unique_iop=''):
...
     # 1. Custom classification from context
     if series_desc:
         classification_dict = get_custom_classification(series_desc, '/flywheel/v0/config.json')
         if classification_dict:
             log.info('Custom classification from config: %s', classification_dict)
 
-    # 2. Classification from SeriesDescription
+    # 2. Classification from Acquisition Label
+    if not classification_dict :
+    	classification_dict = infer_classification(acquisition.label.replace('.dicom.zip',''))
```
Comment (NPann) [inline MR_classifier.py:656]: Not sure what are all the variation you need to capture here and whether or not it applies to acquisition.label you may want to also deal with .dcm and .dcm.zip that I believe are possible variations.

**Review by NPann — APPROVED** [review]

File: MR_classifier.py (hunk 1 above)
Comment (jiavila) [reply MR_classifier.py:656]: @mhedehus see comment

File: MR_classifier.py [hunk 2]
```diff
@@ -157,16 +169,30 @@ def is_diffusion(label):
...
+        #re.compile('_ADC$', re.IGNORECASE),
+        #re.compile('_TRACEW$', re.IGNORECASE),
+        #re.compile('_ColFA$', re.IGNORECASE),
+        #re.compile('_FA$', re.IGNORECASE),
+
+        re.compile('_ADC', re.IGNORECASE),
+        re.compile('_TRACE', re.IGNORECASE),
+        re.compile('_ColFA', re.IGNORECASE),
+        re.compile('_FA_', re.IGNORECASE),
         re.compile('_EXP$', re.IGNORECASE)
```
[the remark this answers is not in the document]
Comment (jiavila) [reply MR_classifier.py:184]: @mhedehus see comment

[the remark this answers is not in the document]
Comment (mhedehus) [reply MR_classifier.py:184]: I don't have strong feelings about that one, as I've not seen the "EXP" tag in any of my data.

File: MR_classifier.py (hunk 1 above)
Comment (mhedehus) [reply MR_classifier.py:656]: Isn't "dicom.zip" added by the CLI during upload, so there should be a/one standard nomenclature?

File: MR_classifier.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (NPann) [reply MR_classifier.py:184]: Up to you. All fine keeping it like this.

File: MR_classifier.py (hunk 1 above)
Comment (NPann) [reply MR_classifier.py:656]: I think that was only on the file when zipping the dicom directory. Looks like this is applied to the acquisition.label here, though. I don't know about the other possibilities, just wanted to raise you awarness.

**Pull request merged**